feat: improve the website with AI 🤖 (#495)
countr/website: content/docs/05-update-cycles.md

# Update Cycles

> At Countr, we're committed to providing you with the best possible counting experience. That's why we update Countr every week with new features, bug fixes, and improvements.

Reviewer: ```suggestion
Countr is committed to providing you with the best possible counting experience. That's why we update Countr every week with new features, bug fixes, and improvements.
```
